app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+  end
+
+  def create
```

Comment:
This isn't the most RESTful approach to creating a record in the joins table. Consider you need to add a form to create a movie. Since you are using the create action to set up a movie-actor relationship, it is already taken and you have to create a movie in a different action. You're correct that it should be a `create` action, but where else could you put this?